db/migrate/20191126181009_seed_test_results.rb

```diff
@@ -0,0 +1,63 @@
+class SeedTestResults < ActiveRecord::Migration[5.2]
+  def up
+    migrated = 0
+    ProfileHost.find_each do |profile_host|
+      host = profile_host.host
+      profile = profile_host.profile
+      test_result = TestResult.create(
+        host: host,
+        profile: profile
+      )
```

Comment:
This looks like it may be the issue. We're creating only one `TestResult` per `ProfileHost`, but there will likely be more than one `TestResult` for each combination of profile and host as new scans are performed for the same profile/host combination.

Reply:
Let's move this `.create` to the `rule_results_by_start_time` block. That would allow us to create 1 test result per group of rule_results in a host/profile group.